packages/shared-business/src/components/CountryPicker.tsx

```diff
@@ -18,44 +15,38 @@ type Props<T extends CountryCCA3> = {
   hideErrors?: boolean;
 };
 
+const removeDuplicated = <T,>(items: T[]): T[] => {
+  const set = new Set(items);
+  return Array.from(set);
+};
+
 export function CountryPicker<T extends CountryCCA3>({
   onValueChange,
   value,
-  items,
+  countries,
   readOnly,
   id,
   error,
   placeholder,
   disabled,
   hideErrors,
 }: Props<T>) {
-  const countries = useMemo(() => {
-    const hasIntl = "Intl" in window && "DisplayNames" in window.Intl;
-    const countryResolver =
-      hasIntl && Intl.DisplayNames.supportedLocalesOf([locale.language]).length
-        ? new Intl.DisplayNames([locale.language], { type: "region" })
-        : undefined;
-    const seen = new Set();
-    return items
-      .filter(item => {
-        const hasBeenSeen = seen.has(item.cca3);
-        seen.add(item.cca3);
-        return !hasBeenSeen;
-      })
-      .map(country => ({
-        name: countryResolver?.of(country.cca2 ?? "") ?? country.name,
-        icon: <Flag width={14} icon={country.cca3} />,
-        value: country.cca3,
+  const items = useMemo(() => {
+    return removeDuplicated(countries)
```

Comment:
tip: to dedup an array you can use `array.filter((item, index, array) => array.indexOf(item) === index)`